app/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+from django.urls import path, include
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
The `include` function is used with a `namespace` argument, but the `cinema.urls` module must define an `app_name` variable for this to work correctly. Ensure that `cinema/urls.py` includes `app_name = 'cinema'` at the top of the file.